addons/account/models/account_tax.py

```diff
@@ -60,6 +60,7 @@ class AccountTaxGroup(models.Model):
              "If not set, the tax group will be displayed after the 'Untaxed amount' subtotal.",
         translate=True,
     )
+    label = fields.Char(string='Label')
```

Comment:
hi :slightly_smiling_face: sorry, just a clarifying question - if this is a label only used for PoS invoices, shouldn't this be done in `point_of_sale` module? :thinking:

Reply:
Hi👋...I have gone with implementing it in the Accounting module for simplicity. Let me explain
So to add a label to an existing localisation, U will just add a new column `label` in the .csv file such as [this one](addons/l10n_se/data/template/account.tax.group-se.csv) and add the respective letter for each tax group. That's it

Adding it to the PoS module which was the initial approach imposed some unnecessary complexity IMHO. but of course let me know if you have another opinion or a better approach!

Thanks😊